querying/github.go

```diff
@@ -137,12 +143,12 @@ func (gh *GithubDataSource) CollectFindings(projects *ProjectCollection, wg *syn
 	return nil
 }
 
-func (gh *GithubDataSource) processRepoFindings(projects *ProjectCollection, repo orgRepo, repoCursor *githubv4.String) error {
+func (gh *GithubDataSource) processRepoFindings(projects *ProjectCollection, repo orgRepo, endCursor githubv4.String) error {
```

Comment:
🤔 Is it even necessary to pass the `endCursor` in now with this new structure? You should be able to access this directly with `repo.VulnerabilityAlerts.PageInfo.EndCursor`. That should work... right?

Reply:
yeah, you are RIGHT, totally my bad haha

Reply:
Fixed!